bench/hermes/optimized/APFloat.cpp.ll

```diff
@@ -10702,15 +10707,15 @@ while.end.loopexit.split.loop.exit70.i:           ; preds = %land.rhs.i
   %tobool.not.i.i.i.i.i.i.i = icmp eq i32 %91, %96
```

Comment:
Not directly related, but there's a missed optimization opportunity here: https://github.com/llvm/llvm-project/issues/88231